frontend/test/metabase/scenarios/dashboard/dashboard-drill.cy.spec.js

```diff
@@ -257,89 +257,76 @@ describe("scenarios > dashboard > dashboard drill", () => {
       .within(() => cy.findByText("foo"));
   });
 
-  it("should pass multiple filters for numeric column on drill-through (metabase#13062)", () => {
-    // Preparation for the test: "Arrange and Act phase" - see repro steps in #13062
-    // 1. set "Rating" Field type to: "Category"
+  ["field value", "card title"].forEach(testCase => {
+    it(`should pass multiple filters for numeric column on drill-through the ${testCase} (metabase#13062)`, () => {
+      // Set "Rating" Field type to: "Category"
+      cy.request("PUT", `/api/field/${REVIEWS.RATING}`, {
+        semantic_type: "type/Category",
+      });
 
-    cy.request("PUT", `/api/field/${REVIEWS.RATING}`, {
-      semantic_type: "type/Category",
-    });
-    // 2. create a question based on Reviews
-    cy.request("POST", `/api/card`, {
-      name: "13062Q",
-      dataset_query: {
-        database: 1,
+      const questionDetails = {
```

Comment:
Seems like this const could be declared in the `describe`, before the blocks with `it`, to be reused.